app/src/Entity/Appointment.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use App\Repository\AppointmentRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: AppointmentRepository::class)]
+#[ApiResource]
+class Appointment
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private int $id;
+
+    #[ORM\ManyToOne(inversedBy: 'appointments')]
+    private User $user_id;
+
+    #[ORM\Column(length: 255)]
+    private string $title;
+
+    #[ORM\Column(type: Types::TEXT, nullable: true)]
+    private ?string $description = null;
+
+    #[ORM\Column]
+    private \DateTimeImmutable $schedule;
+
+    #[ORM\Column(type: Types::DECIMAL, precision: 10, scale: '0')]
+    private string $price;
+
+    #[ORM\Column(length: 255)]
+    private string $status;
+
+    #[ORM\Column]
+    private ?\DateTimeImmutable $created_at = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?\DateTimeImmutable $updated_at = null;
```

Comment:
Remove backslash here -> import DateTimeImmutable via use statement.

Use `camelCase` with variable names. Change this to `$createdAt`.